app/harvesters/Itms/harvester.rb

```diff
@@ -15,7 +15,112 @@ def initialize(min_id, downloader)
 
   def load_and_parse_endpoint(endpoint)
     response = downloader.get(endpoint)
-    raise(::Itms::Harvester::ApiError, 'API not available') if response.response_code != 200
-    JSON.parse(response.body)
+    handle_api_response(response)
+  end
+
+  def handle_api_response(response)
+    case response.response_code
+    when 200
+      JSON.parse(response.body)
+    when 404
+      []
```

Comment:
v tom pripade to vratim do povodneho stavu.

tj, https://github.com/slovensko-digital/harvester.ecosystem/pull/8#issuecomment-307563307 sa aktualne implementovat nebude.